INDLinkLabel.swift

```diff
@@ -141,17 +149,24 @@ import UIKit
             if let URL = URL {
                 let glyphRange = self.layoutManager.glyphRangeForCharacterRange(range, actualCharacterRange: nil)
                 ranges.append(LinkRange(URL: URL, glyphRange: glyphRange))
-                
```

Comment:
fixed in https://github.com/cdzombak/INDLinkLabel/commit/a4f457b585d7f3f821135db5b8c69acaf3938711